test/test_archivesync.py

```diff
@@ -48,19 +49,31 @@ def test_check_wal_archive_integrity(requests_put_mock, requests_head_mock):
     # Check integrity within same timeline
     arsy.get_current_wal_file = Mock(return_value="00000005000000000000008F")
     arsy.get_first_required_wal_segment = Mock(return_value="00000005000000000000008C")
-    assert arsy.check_wal_archive_integrity() == 0
+    assert arsy.check_wal_archive_integrity(False) == 0
```

Comment:
Cosmetic thing but might want to use kwarg here as in new_backup_on_failure=False so it's self-evident what we're passing in.